jvm-packages/jenkins/Jenkinsfile-Release

```diff
@@ -268,6 +271,36 @@ try {
         }
     }
 
+    if (params.publicRelease) {
+        stage('Git Release') {
+            node('master') {
+                pullDockerImage(UBUNTU_DOCKER_IMAGE)
+                docker.withRegistry("https://${REGISTRY}") {
+                    docker.image(UBUNTU_DOCKER_IMAGE).inside("--init -v /home/jenkins/.ssh:/home/jenkins/.ssh:ro -v /home/jenkins/.gitconfig:/home/jenkins/.gitconfig:ro") {
```

Comment:
Would it be a good idea to have the keys in Jenkins?

Reply:
I would be careful, it might cause some troubles. I would prefer to do releases from master only (at least the tasks which require ssh auth).